### ISSUE TITLE
The command to clear the cache

### ISSUE THREAD

**Issue by soyv4** [lllyasviel/stable-diffusion-webui-forge#1433]
Greetings! tell me if it makes sense to write a command in the webui-user batnik to clear the cache of garbage? 
del /q "tmp\tmp*" >nul 2>nul & del /q "tmp\gradio\*.*" >nul 2>nul & rd /s /q "pip\cache" >nul 2>nul & for /d %%x in (tmp\tmp*,tmp\pip*,tmp\gradio\*) do (rd /s /q "%%x" 2>nul || ("%%x" && exit /b 1))
![1111](https://github.com/user-attachments/assets/b3fffe9f-7dda-486d-836e-e84ace6bf489)

**Comment by JiggyDancer** [comment]
Yes please, was looking for that option while trying out TeaCache.
